noir-projects/noir-contracts/contracts/avm_test_contract/src/main.nr

```diff
@@ -191,6 +191,15 @@ pub contract AvmTest {
         commitment
     }
 
+    #[public]
+    fn conditional_move(x: [Field; 1], y: [Field; 1], b: bool) -> [Field; 1] {
```

Comment:
Not sure this actually triggers a CMOV, but there's no other way to test this given our current infra :)